server/cli/flags.go

```diff
@@ -87,6 +87,10 @@ func initFlags(ctx *server.Context) {
 		"--scan_interval to adjust the target for the duration of a single scan "+
 		"through a store's ranges. The scan is slowed as necessary to approximately"+
 		"achieve this duration.")
+
+	flag.DurationVar(&ctx.MetricsFrequency, "metrics-frequency", ctx.MetricsFrequency, "specify "+
+		"--metrics-frequency to adjust the frequency at which the server records "+
```

Comment:
Could you edit this instance and all others in the documentation here to use a single hyphen in front of command line flags? Most other places in the code we've been trying to be consistent about using a single hyphen. Let's choose something and stick to it.